packages/extension-chakra-storefront/config/default.json

```diff
@@ -1,50 +1,144 @@
 {
     "enabled": true,
     "activeDataEnabled": false,
+    "maxCacheAge": 900,
+    "staleWhileRevalidate": 900,
```

Comment:
Does it make more sense to add these into the Commerce API setup?

Reply:
I don't think it makes sense to put it in the commerce api since these set up has nothing to do with commerce ap. They are not for React query either. We are using them to cache the `Cache-control` for the express app in general. I think it makes sense for them to be here. See [here](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2174/files#diff-51a19f3e5d42d40f70cf6482a4194c9af03ea37f22445a35c2a0ec7078fff014R63)